### ISSUE TITLE
any tuning suggestions? best FID i've achieved on celeba 128x128 is ~40

### ISSUE THREAD

**Issue by GetsEclectic** [lucidrains/stylegan2-pytorch#111]
Seems like I should be able to get down to a FID of 3-6 with this architecture given the results on FFHQ, LSUN Cat, etc.

I got to ~40 at 120k iterations with this config:
bin/stylegan2_pytorch --data ../imgs/ --image-size 128 --batch_size 32 --gradient_accumulate_every 1 --fp16 --ttur_mult 1

Things I've tried to improve the score so far:

- reducing network capacity to 8
- applying gradient penalty every step instead of every 4 steps
- increasing learning rate to 1e-3 and clipping gradients

I'm a total beginner so any suggestions would be great, thanks!

**Comment by GetsEclectic** [comment]
Tried switching to 256x256 and I got to ~50 FID at 50k iterations. Also I got a roughly 4x slowdown when pl regularization kicked in and had to disable it.

```
bin/stylegan2_pytorch --data ../imgs/ --image-size 256 --batch_size 8 --gradient_accumulate_every 2 --ttur_mult 1
```
